go.mod

```diff
@@ -5,8 +5,8 @@ go 1.23
 toolchain go1.23.0
 
 require (
-	go.opentelemetry.io/otel v1.31.0
-	go.opentelemetry.io/otel/trace v1.31.0
+	go.opentelemetry.io/otel v1.32.0
+	go.opentelemetry.io/otel/trace v1.32.0
```

Comment:
I found this open bug for the version v1.32.0. @elliehastings @gffking do you think it could impact maya-server?

https://github.com/open-telemetry/opentelemetry-go/issues/5976

Reply:
We're not using the OpenTelemetry gRPC exporter, so I think we should be fine.